plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -384,7 +396,7 @@ private Optional<Table> loadTable(HiveTableName hiveTableName)
     @Override
     public PartitionStatistics getTableStatistics(Table table)
     {
-        return get(tableStatisticsCache, hiveTableName(table.getDatabaseName(), table.getTableName()));
+        return get(tableStatisticsCache, hiveTableName(table.getDatabaseName(), table.getTableName()), () -> delegate.getTableStatistics(table));
```

Comment:
Is this related to "Disallow using of single-entry loader in bulk cache"?

(It may seem to a reader that the commit is a result of two separate changes squashed together)

Reply:
> Is this related to "Disallow using of single-entry loader in bulk cache"?

It is, since single-entry loader (in `LoadincCache`) is not allowed, we need to provide explicit loader here.

Reply:
the `tableStatisticsCache` is not bulk, is it?

Reply:
Moved to separate commit, see also https://github.com/trinodb/trino/pull/15838/commits/65531e0f6b2dae94912f29dbc716614d93b94473#r1087611670 which this change is useful in general